test/enum.c

```diff
@@ -729,61 +708,114 @@ test_funcs(void)
 error:
     H5E_BEGIN_TRY
     {
-        H5Tclose(type);
+        H5Tclose(tid);
     }
     H5E_END_TRY;
     return 1;
 }
 
 /*-------------------------------------------------------------------------
- * Function:	main
- *
- * Purpose:
+ * Function:    test_copying_empty_enum
  *
- * Return:	Success:
- *
- *		Failure:
- *
- * Programmer:	Robb Matzke
- *              Tuesday, December 22, 1998
+ * Purpose:     Test that copying an empty enum works, including implicitly
+ *              when copying compound datatypes containing empty enums
  *
+ * Return:      Success:    0
+ *              Failure:    1
  *-------------------------------------------------------------------------
  */
+static int
+test_compound_insert_empty_enum(void)
```

Comment:
This test ensures that copying empty enums works, whether directly or as a part of a compound type.